.github/workflows/CD.yml

```diff
@@ -31,3 +31,39 @@ jobs:
           user: accessnri
           label: main
           token: ${{ secrets.anaconda_token }}
+
+  pypi-build:
+    name: Build package for PyPI
+    if: github.repository == 'payu-org/payu' && github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
This can now be v4 (also above)

Reply:
```suggestion
      - uses: actions/checkout@v4
```

Reply:
Hrm. That is annoying, I can't batch commit my own suggestions.

Reply:
@dougiesquire can you batch up and commit my suggestions? Not being lazy (am), but also curious if that is the problem, that they're my suggestions.